src/runtime-rs/crates/hypervisor/src/ch/inner_hypervisor.rs

```diff
@@ -676,7 +677,16 @@ impl CloudHypervisorInner {
     }
 
     pub(crate) async fn get_thread_ids(&self) -> Result<VcpuThreadIds> {
-        Ok(VcpuThreadIds::default())
+        let mut vcpus = HashMap::new();
+
+        let vcpu = 0;
```

Comment:
@jodh-intel is this always 0?

Reply:
@cmaf - well, I'm not sure what else we could do here. Thoughts?

Reply:
@jodh-intel This api is meant to return all the tids corresponding to all the vcpus that the VM uses. 

qemu has an qmp api `query-cpus-fast` that returns the details about the vcpus: https://www.qemu.org/docs/master/system/cpu-hotplug.html

I am not sure if such an api exists or has been added to clh since the go implementation. 
cc @likebreath  Can you let us know about this?

Else we may have to follow the same method that we used for the go-implementation i.e walk through the /proc/pid  tree:
https://github.com/kata-containers/kata-containers/blob/main/src/runtime/virtcontainers/clh.go#L760

Reply:
> qemu has an qmp api `query-cpus-fast` that returns the details about the vcpus: https://www.qemu.org/docs/master/system/cpu-hotplug.html
> 
> I am not sure if such an api exists or has been added to clh since the go implementation. cc @likebreath Can you let us know about this?

@amshinde is right. Cloud Hypervisor does not provide such pid information at this time. Of course, please feel free to send feature requests to the CLH community.

One clarification questions: what kind of PID information does kata want from VMM?